config/routes.rb

```diff
@@ -383,6 +383,7 @@
   post 'comment/react/update/:id' => 'comment#react_update'
 
   get 'topic-tree' => 'tag#topic_tree'
+  get 'tags/:subtopic' => 'tag#subtopics'
```

Comment:
```suggestion
  get 'tags/subtopic/:topic' => 'tag#subtopics'
```
Here, Rails routing takes some getting used to - the parts with `:symbol`-style prefixes will capture anything written there in the URL, while all other text becomes literally part of the URL: (like `tags/subtopic/dinosaurs' will match `tags/subtopic/` but then will capture `dinosaurs` and create a variable called `params[:dinosaurs]` which you will then be able to reference from the controller method).

By setting `:topic` here, we give a useful name to the parameter it will capture from the URL.